src/smokeTest/java/uk/gov/hmcts/reform/bulkscanprocessor/WhenApplicationDeployedTest.java

```diff
@@ -18,7 +18,7 @@
 @TestPropertySource("classpath:application.yaml")
 public class WhenApplicationDeployedTest {
 
-    @Value("${test-url}")
+    @Value("http://${test-host}")
```

Comment:
an env var with:
TEST_URL should be exported to you and it will have https://full-url here
no need to change this